enumerables.rb

```diff
@@ -0,0 +1,109 @@
+module Enumerable
+
+  def my_each
+    if block_given?
+      (0..length - 1).each do |i|
+        yield(self[i])
+      end
+      self
+    else
+      Enumerable
+    end
+  end
```

Comment:
Hi, I just tested my method again and it returns true and they work the same, please advise
 further, I do not understand what else to do because I am getting same results

Reply:
Should I not use each method in my my_each, rather use while loop? please advise